### PR TITLE
Closes #2502: Update Analytics Dashboard link in toolbar.

### LINKED ISSUE #2502: update data studio dashboard link
Analytics Dashboard link in toolbar needs to be updated to point to new GA4 dashboard.

## Conditions of satisfaction
- [ ] Should ‹testable condition that should be satisfied›
- [ ] Should ‹testable condition that should be satisfied›
- [ ] …

### PR DESCRIPTION
**Pull request by Krystation**
## Related issues

Closes #2502

## How to test
Log in as content admin or admin or content editor
Click the Quickstart link in toolbar
Click the analytics dashboard link described here:  https://quickstart.arizona.edu/site-admin/analytics
See if the link takes you to the correct place.

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
   - [ ] Bug fix
   - [ ] Accessibility, performance, or security improvement
   - [ ] Critical institutional link or brand change
   - [ ] Adding experimental module
   - [ ] Update experimental module
- **Minor release changes**
   - [ ] New feature
   - [ ] Breaking or visual change to existing behavior
   - [ ] Upgrade experimental module to stable
   - [ ] Enable existing module by default or database update
   - [ ] Non-critical brand change
   - [ ] New internal API or API improvement with backwards compatibility
   - [ ] Risky or disruptive cleanup to comply with coding standards
   - [ ] High-risk or disruptive change (requires upgrade path, risks regression, etc.)
- **Other or unknown**
   - [ ] Other or unknown

### Drupal core
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch level release (non-security bug-fix release)
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major or minor level update
- **Other or unknown**
   - [ ] Other or unknown

### Drupal contrib projects
- **Patch release changes**
   - [ ] Security update
   - [ ] Patch or minor level update
   - [ ] Add new module
   - [ ] Patch removal that's no longer necessary
- **Minor release changes**
   - [ ] Major level update
- **Other or unknown**
   - [ ] Other or unknown

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by trackleft — APPROVED** [review]
looks correct to me

**Comment by joeparsons** [comment]
Question: should this get backported to Quickstart 2.6 / 2.5?

**Review by RyanDool — APPROVED** [review]

**Comment by camikazegreen** [comment]
> Question: should this get backported to Quickstart 2.6 / 2.5?

Yes, I think that it should. I think it falls under this patch release category:
Critical institutional link changes

FWIW, the probo site for this PR is acting very strange. I can't see how it could be related to this PR, but I can't put my checkmark on with these issues:
- It is much slower than usual, especially any attempts to login have been unsuccessful
- The main navigation is messed up. Extra spacing above the menu items and heavier font-weight

File: modules/custom/az_google_tag/az_google_tag.module [hunk 1]
```diff
@@ -15,7 +15,7 @@ function az_google_tag_toolbar_alter(&$items) {
     $site_url = \Drupal::request()->getHost();
     $items['az_quickstart']['tray']['az_links']['#links']['analytics'] = [
       'title' => t('Analytics Dashboard'),
-      'url' => Url::fromUri('https://datastudio.google.com/u/0/reporting/938c3f2a-6dbf-4968-9edd-8963450b4200/page/kTLWB', [
+      'url' => Url::fromUri('https://lookerstudio.google.com/reporting/ef02c272-afe4-4862-b4bd-8706e3436b2f', [
         'query' => [
           'params' => '{"df62":"include%EE%80%800%EE%80%80IN%EE%80%80' . $site_url . '/"}',
```
Comment (joeparsons) [inline modules/custom/az_google_tag/az_google_tag.module:20]: The existing version of this link in Quickstart (that links to the old analytics dashboard) automatically pre-selects the correct subdomain based on the domain of the site that rendered the link.

I tested replacing the ProboCI site domain in the generated link with `quickstart.arizona.edu` but found that it did not pre-select the correct subdomain as expected.

I think if we make these changes to the URL the subdomain auto selection functionality would work as expected (on sites with real subdomains -- not ProboCI).

```suggestion
      'url' => Url::fromUri('https://lookerstudio.google.com/reporting/ef02c272-afe4-4862-b4bd-8706e3436b2f/page/nbaTD', [
        'query' => [
          'params' => '{"df21":"include%EE%80%800%EE%80%80IN%EE%80%80' . $site_url . '/"}',
```
Example generated link following this pattern:
https://lookerstudio.google.com/u/0/reporting/ef02c272-afe4-4862-b4bd-8706e3436b2f/page/nbaTD?params=%7B%22df21%22:%22include%25EE%2580%25800%25EE%2580%2580IN%25EE%2580%2580quickstart.arizona.edu%22%7D

**Review by joeparsons — APPROVED** [review]

**Comment by Carris-Toupin** [comment]
I signed in as Admin, Content Admin and Content Editor and the QS2 > Analytics Dashboard link takes me to the Universal GA4 page for U of AZ for all 3 roles.

**Review by Carris-Toupin — APPROVED** [review]
I signed in as Admin, Content Admin and Content Editor and the QS2 > Analytics Dashboard link takes me to the Universal GA4 page for U of AZ for all 3 roles.

**Review by trackleft — APPROVED** [review]

**Pull request merged**